server.go

```diff
@@ -64,12 +64,17 @@ func handleComments(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 
-		// Write out the json response
-		commentsEncoder := json.NewEncoder(cFile)
-		commentsEncoder.Encode(comments)
+		// Pretty print the JSON, write to file
+		var encodedComments, _ = json.MarshalIndent(comments, "", "    ")
```

Comment:
Can you? That would be awesome. I got my fill of writing Go for the time being :D

The servers implementations are by far the least important part of the whole thing here. The variety is intended to make sure anybody can run *something* without having to muck around too much with installing different thing (eg, shipping just a Go server would be bad). Keeping the behavior consistent across these servers is important to me though because it makes it easier to help people. No saying "oh you're using the Go server and it does things slightly differently... normally you wouldn't have to parse a single long line of JSON in your head".

So if you wouldn't mind doing that, I'd appreciate it. If you wanted to add comments saying that "*normally* you wouldn't do this, you would just do X, Y, Z" instead, that's totally fine.